views/products/technical.tpl

```diff
@@ -122,6 +122,9 @@
         % else:
             {{ code.get('type', 'url') }}
         % end
+        % if 'date_last_commit' in code:
+          - last commit: {{ code['date_last_commit'].strftime('%Y-%m-%d')}}
```

Comment:
```suggestion
          - last commit: {{ code['date_last_commit'].date()}}
```
The default string formatting of a `date` is [conveniently](https://docs.python.org/3/library/datetime.html#datetime.date.__str__) the ISO format. :)

Reply:
Dang it stackoverflow - I pay you nothing and you give me these crappy suggestions...